libstuff/libstuff.h

```diff
@@ -97,6 +99,23 @@ class STableComp : binary_function<string, string, bool> {
     };
 };
 
+#define STHROW(_MSG_) throw SException(_MSG_, __FILE__, __LINE__)
+#define STHROW_STACK(_MSG_) throw SException(_MSG_, __FILE__, __LINE__, true)
```

Comment:
This is unused... do we need it? 😕

Reply:
I don't mind removing the `#define` for it. I'm reluctant to remove the code that supports it in `SException` because I expect it will come in handy whenever we next try to debug an uncaught exception.